.drone.star

```diff
@@ -13,14 +14,53 @@ def main(ctx):
 
     # Version branches never deploy themselves, but instead trigger a deployment in deployment_branch
     # This must not be changed in version branches
-    deployment_branch = "master"
+    deployment_branch = default_branch
+    pdf_branch = default_branch
```

Comment:
As described in the master branch PR, `pdf_branch` need to be changed on version branches to `pdf_branch = base_branch`